src/JoomlaBrowser.php

```diff
@@ -129,16 +129,14 @@ public function installJoomla()
         $this->debug('I wait for Main Configuration');
         $I->waitForElement('#jform_language', 10);
         // Wait for chosen to render the field
-        $I->wait(1);
```

Comment:
The removal of this line caused the browser to get stuck during installation:

<img width="685" alt="screen shot 2016-01-15 at 11 57 11" src="https://cloud.githubusercontent.com/assets/1375475/12351617/27800504-bb7f-11e5-8fe1-f534e57c36bd.png">

I tested running the tests with JoomlaBrowser in the commit previous to this one and it works:

<img width="439" alt="screen shot 2016-01-15 at 11 57 45" src="https://cloud.githubusercontent.com/assets/1375475/12351623/373c3df0-bb7f-11e5-8266-f2cc4a9d6b90.png">

I'm sending the fix right now.